lib/helpers.js

```diff
@@ -62,10 +62,30 @@ helpers.streamToChannel = event => {
     return out;
 };
 
-helpers.objectToSchema = obj => {
+const objectToSchema = obj => {
```

Comment:
Because this function previously only used in the duplicated block code, is it okay to make it as const since now they are in the same file?

Reply:
yeap, very good